### ISSUE TITLE
You have macOS 10.15.7. The application requires macOS 11.0 or later.

### ISSUE THREAD

**Issue by glensc** [docker/for-mac#6732]
Seems docker for mac auto-updated to version that is no supported.

  - [x] I have tried with the latest version of Docker Desktop
  - [ ] I have tried disabling enabled experimental features
  - [ ] I have uploaded Diagnostics
  - Diagnostics ID:

### Expected behavior

It should have not updated if the new version can't run on this platform

### Actual behavior

You have macOS 10.15.7. The application requires macOS 11.0 or later. 

and now I'm stuck with no docker at all.

### Information

  - macOS Version: 10.15.7
  - Intel chip or Apple chip: Intel
  - Docker Desktop Version: how to obtain version if app is not starting?

### Output of `/Applications/Docker.app/Contents/MacOS/com.docker.diagnose check`

```
~ ➜ /Applications/Docker.app/Contents/MacOS/com.docker.diagnose check
Starting diagnostics

[PASS] DD0027: is there available disk space on the host?
[PASS] DD0028: is there available VM disk space?
[PASS] DD0018: does the host support virtualization?
[FAIL] DD0001: is the application running? Docker is not running
[PASS] DD0017: can a VM be started?
[PASS] DD0016: is the LinuxKit VM running?
[FAIL] DD0011: are the LinuxKit services running? failed to ping VM diagnosticsd with error: Get "http://ipc/ping": dial unix diagnosticd.sock: connect: no such file or directory
[2023-02-10T14:10:50.769593000Z][com.docker.diagnose][I] ipc.NewClient: ba97c852-diagnose -> diagnosticd.sock diagnosticsd
[common/pkg/diagkit/gather/diagnose.glob..func12()
[	common/pkg/diagkit/gather/diagnose/linuxkit.go:18 +0x8c
[common/pkg/diagkit/gather/diagnose.(*test).GetResult(0x100d638a0)
[	common/pkg/diagkit/gather/diagnose/test.go:46 +0x43
[common/pkg/diagkit/gather/diagnose.Run.func1(0x100d638a0)
[	common/pkg/diagkit/gather/diagnose/run.go:17 +0x5a
[common/pkg/diagkit/gather/diagnose.walkOnce.func1(0x4?, 0x100d638a0)
[	common/pkg/diagkit/gather/diagnose/run.go:142 +0x77
[common/pkg/diagkit/gather/diagnose.walkDepthFirst(0x3, 0x100d638a0, 0xc000715720)
[	common/pkg/diagkit/gather/diagnose/run.go:151 +0x87
[common/pkg/diagkit/gather/diagnose.walkDepthFirst(0x2, 0x100d63920, 0xc000715720)
[	common/pkg/diagkit/gather/diagnose/run.go:148 +0x52
[common/pkg/diagkit/gather/diagnose.walkDepthFirst(0x1, 0x100d63a20, 0xc000715720)
[	common/pkg/diagkit/gather/diagnose/run.go:148 +0x52
[common/pkg/diagkit/gather/diagnose.walkDepthFirst(0x0, 0x100d63ca0, 0xc000715720)
[	common/pkg/diagkit/gather/diagnose/run.go:148 +0x52
[common/pkg/diagkit/gather/diagnose.walkOnce(0x100710580?, 0xc00059f888)
[	common/pkg/diagkit/gather/diagnose/run.go:137 +0xcc
[common/pkg/diagkit/gather/diagnose.Run(0x100d63ca0, 0xc00059fab0?, {0xc00059fb18, 0x1, 0x1})
[	common/pkg/diagkit/gather/diagnose/run.go:16 +0x1d4
[main.checkCmd({0xc0000d6010?, 0x6?, 0x4?}, {0x0, 0x0})
[	common/cmd/com.docker.diagnose/main.go:138 +0x105
[main.main()
[	common/cmd/com.docker.diagnose/main.go:102 +0x29f
[2023-02-10T14:10:50.772242000Z][com.docker.diagnose][I] (b9863f92) ba97c852-diagnose C->S diagnosticsd GET /ping
[2023-02-10T14:10:50.781502000Z][com.docker.diagnose][W] (b9863f92) ba97c852-diagnose C<-S NoResponse GET /ping (9.195016ms): Get "http://ipc/ping": dial unix diagnosticd.sock: connect: no such file or directory

[FAIL] DD0004: is the Docker engine running? Get "http://ipc/docker": dial unix lifecycle-server.sock: connect: no such file or directory
[2023-02-10T14:10:50.783666000Z][com.docker.diagnose][I] ipc.NewClient: a2890bfc-com.docker.diagnose -> lifecycle-server.sock VMDockerdAPI
[linuxkit/pkg/desktop-host-tools/pkg/client.NewClientForPath(...)
[	linuxkit/pkg/desktop-host-tools/pkg/client/client.go:63
[linuxkit/pkg/desktop-host-tools/pkg/client.NewClient({0x1007d6593, 0x13})
[	linuxkit/pkg/desktop-host-tools/pkg/client/client.go:57 +0x99
[common/pkg/diagkit/gather/diagnose.isDockerEngineRunning()
[	common/pkg/diagkit/gather/diagnose/dockerd.go:21 +0x29
[common/pkg/diagkit/gather/diagnose.(*test).GetResult(0x100d63920)
[	common/pkg/diagkit/gather/diagnose/test.go:46 +0x43
[common/pkg/diagkit/gather/diagnose.Run.func1(0x100d63920)
[	common/pkg/diagkit/gather/diagnose/run.go:17 +0x5a
[common/pkg/diagkit/gather/diagnose.walkOnce.func1(0x3?, 0x100d63920)
[	common/pkg/diagkit/gather/diagnose/run.go:142 +0x77
[common/pkg/diagkit/gather/diagnose.walkDepthFirst(0x2, 0x100d63920, 0xc000715720)
[	common/pkg/diagkit/gather/diagnose/run.go:151 +0x87
[common/pkg/diagkit/gather/diagnose.walkDepthFirst(0x1, 0x100d63a20, 0xc000715720)
[	common/pkg/diagkit/gather/diagnose/run.go:148 +0x52
[common/pkg/diagkit/gather/diagnose.walkDepthFirst(0x0, 0x100d63ca0, 0xc000715720)
[	common/pkg/diagkit/gather/diagnose/run.go:148 +0x52
[common/pkg/diagkit/gather/diagnose.walkOnce(0x100710580?, 0xc00059f888)
[	common/pkg/diagkit/gather/diagnose/run.go:137 +0xcc
[common/pkg/diagkit/gather/diagnose.Run(0x100d63ca0, 0xc00059fab0?, {0xc00059fb18, 0x1, 0x1})
[	common/pkg/diagkit/gather/diagnose/run.go:16 +0x1d4
[main.checkCmd({0xc0000d6010?, 0x6?, 0x4?}, {0x0, 0x0})
[	common/cmd/com.docker.diagnose/main.go:138 +0x105
[main.main()
[	common/cmd/com.docker.diagnose/main.go:102 +0x29f
[2023-02-10T14:10:50.786255000Z][com.docker.diagnose][I] (03933f39) a2890bfc-com.docker.diagnose C->S VMDockerdAPI GET /docker
[2023-02-10T14:10:50.787899000Z][com.docker.diagnose][W] (03933f39) a2890bfc-com.docker.diagnose C<-S NoResponse GET /docker (1.627629ms): Get "http://ipc/docker": dial unix lifecycle-server.sock: connect: no such file or directory
[2023-02-10T14:10:50.788963000Z][com.docker.diagnose][I] (03933f39-1) a2890bfc-com.docker.diagnose C->S VMDockerdAPI GET /ping
[2023-02-10T14:10:50.790392000Z][com.docker.diagnose][W] (03933f39-1) a2890bfc-com.docker.diagnose C<-S NoResponse GET /ping (1.415927ms): Get "http://ipc/ping": dial unix lifecycle-server.sock: connect: no such file or directory
[2023-02-10T14:10:51.791993000Z][com.docker.diagnose][I] (03933f39-2) a2890bfc-com.docker.diagnose C->S VMDockerdAPI GET /ping
[2023-02-10T14:10:51.793522000Z][com.docker.diagnose][W] (03933f39-2) a2890bfc-com.docker.diagnose C<-S NoResponse GET /ping (1.518056ms): Get "http://ipc/ping": dial unix lifecycle-server.sock: connect: no such file or directory
[2023-02-10T14:10:52.794836000Z][com.docker.diagnose][I] (03933f39-3) a2890bfc-com.docker.diagnose C->S VMDockerdAPI GET /ping
[2023-02-10T14:10:52.796385000Z][com.docker.diagnose][W] (03933f39-3) a2890bfc-com.docker.diagnose C<-S NoResponse GET /ping (1.558177ms): Get "http://ipc/ping": dial unix lifecycle-server.sock: connect: no such file or directory
[2023-02-10T14:10:53.798152000Z][com.docker.diagnose][I] (03933f39-4) a2890bfc-com.docker.diagnose C->S VMDockerdAPI GET /ping
[2023-02-10T14:10:53.800083000Z][com.docker.diagnose][W] (03933f39-4) a2890bfc-com.docker.diagnose C<-S NoResponse GET /ping (1.910704ms): Get "http://ipc/ping": dial unix lifecycle-server.sock: connect: no such file or directory
[2023-02-10T14:10:54.802191000Z][com.docker.diagnose][I] (03933f39-5) a2890bfc-com.docker.diagnose C->S VMDockerdAPI GET /ping
[2023-02-10T14:10:54.803806000Z][com.docker.diagnose][W] (03933f39-5) a2890bfc-com.docker.diagnose C<-S NoResponse GET /ping (1.626215ms): Get "http://ipc/ping": dial unix lifecycle-server.sock: connect: no such file or directory
[2023-02-10T14:10:55.805737000Z][com.docker.diagnose][I] (03933f39-6) a2890bfc-com.docker.diagnose C->S VMDockerdAPI GET /ping
[2023-02-10T14:10:55.807361000Z][com.docker.diagnose][W] (03933f39-6) a2890bfc-com.docker.diagnose C<-S NoResponse GET /ping (1.605871ms): Get "http://ipc/ping": dial unix lifecycle-server.sock: connect: no such file or directory
[2023-02-10T14:10:56.809671000Z][com.docker.diagnose][I] (03933f39-7) a2890bfc-com.docker.diagnose C->S VMDockerdAPI GET /ping
[2023-02-10T14:10:56.811235000Z][com.docker.diagnose][W] (03933f39-7) a2890bfc-com.docker.diagnose C<-S NoResponse GET /ping (1.572319ms): Get "http://ipc/ping": dial unix lifecycle-server.sock: connect: no such file or directory
[2023-02-10T14:10:57.813763000Z][com.docker.diagnose][I] (03933f39-8) a2890bfc-com.docker.diagnose C->S VMDockerdAPI GET /ping
[2023-02-10T14:10:57.815399000Z][com.docker.diagnose][W] (03933f39-8) a2890bfc-com.docker.diagnose C<-S NoResponse GET /ping (1.623529ms): Get "http://ipc/ping": dial unix lifecycle-server.sock: connect: no such file or directory

[PASS] DD0015: are the binary symlinks installed?
[FAIL] DD0031: does the Docker API work? Cannot connect to the Docker daemon at unix://docker.raw.sock. Is the docker daemon running?
[PASS] DD0013: is the $PATH ok?
Cannot connect to the Docker daemon at unix:///var/run/docker.sock. Is the docker daemon running?
[FAIL] DD0003: is the Docker CLI working? exit status 1
[FAIL] DD0038: is the connection to Docker working? HTTP GET https://login.docker.com: Get "https://login.docker.com": proxyconnect tcp: dial unix httpproxy.sock: connect: no such file or directory
[FAIL] DD0014: are the backend processes running? 5 errors occurred:
	* com.docker.backend is not running
	* vpnkit-bridge is not running
	* com.docker.vpnkit is not running
	* com.docker.driver.amd64-linux is not running
	* com.docker.hyperkit is not running


[FAIL] DD0007: is the backend responding? failed to ping com.docker.backend with error: Get "http://ipc/ping": dial unix backend.sock: connect: no such file or directory
[2023-02-10T14:11:04.070401000Z][com.docker.diagnose][I] ipc.NewClient: 4f6607bd-diagnose -> backend.sock BackendAPI
[common/pkg/backend.NewClientForPath({0x1007cdd8b?, 0x0?}, {0xc000159ac0?, 0xc000194460?})
[	common/pkg/backend/client.go:176 +0x3d
[common/pkg/backend.NewClient({0x1007cdd8b, 0x8})
[	common/pkg/backend/client.go:171 +0x50
[common/pkg/diagkit/gather/diagnose.glob..func8()
[	common/pkg/diagkit/gather/diagnose/ipc.go:25 +0x25
[common/pkg/diagkit/gather/diagnose.(*test).GetResult(0x100d63d20)
[	common/pkg/diagkit/gather/diagnose/test.go:46 +0x43
[common/pkg/diagkit/gather/diagnose.Run.func1(0x100d63d20)
[	common/pkg/diagkit/gather/diagnose/run.go:17 +0x5a
[common/pkg/diagkit/gather/diagnose.walkOnce.func1(0x2?, 0x100d63d20)
[	common/pkg/diagkit/gather/diagnose/run.go:142 +0x77
[common/pkg/diagkit/gather/diagnose.walkDepthFirst(0x1, 0x100d63d20, 0xc000715720)
[	common/pkg/diagkit/gather/diagnose/run.go:151 +0x87
[common/pkg/diagkit/gather/diagnose.walkDepthFirst(0x0, 0x100d63ca0, 0xc000715720)
[	common/pkg/diagkit/gather/diagnose/run.go:148 +0x52
[common/pkg/diagkit/gather/diagnose.walkOnce(0x100710580?, 0xc00059f888)
[	common/pkg/diagkit/gather/diagnose/run.go:137 +0xcc
[common/pkg/diagkit/gather/diagnose.Run(0x100d63ca0, 0xc00059fab0?, {0xc00059fb18, 0x1, 0x1})
[	common/pkg/diagkit/gather/diagnose/run.go:16 +0x1d4
[main.checkCmd({0xc0000d6010?, 0x6?, 0x4?}, {0x0, 0x0})
[	common/cmd/com.docker.diagnose/main.go:138 +0x105
[main.main()
[	common/cmd/com.docker.diagnose/main.go:102 +0x29f

[... stuck...]
```

### Steps to reproduce the behavior

  1. ...
  2. ...

**Comment by glensc** [comment]
here's some blog how to use minikube without kubernetes, so could continue using docker on mac:
- https://medium.com/rahasak/replace-docker-desktop-with-minikube-and-hyperkit-on-macos-783ce4fb39e3

**Comment by ddelange** [comment]
fwiw, the `<sparkle:minimumSystemVersion>10.15.0</sparkle:minimumSystemVersion>` property needs updating to 11+ as well in the [appcast](https://desktop.docker.com/mac/main/amd64/appcast.xml)

**Comment by glensc** [comment]
Is there a way to downgrade? and since what version this broke?

**Comment by glensc** [comment]
copy of [https://desktop.docker.com/mac/main/amd64/appcast.xml](https://github.com/docker/for-mac/files/10720484/appcast.xml.txt)

**Comment by ddelange** [comment]
last one still compatible with catalina seems to be 4.15.0 https://desktop.docker.com/mac/main/amd64/93002/Docker.dmg

**Comment by ddelange** [comment]
can probably be closed as duplicate of #6701
